src/main/webapp/js/webrtc_adaptor.js

```diff
@@ -1289,31 +1294,35 @@ export class WebRTCAdaptor
 	}
 
 
-	turnOffLocalCamera() 
-	{
-		if (this.remotePeerConnection != null) {
-
-			var track = this.localStream.getVideoTracks()[0];
-			track.enabled = false;
-		}
-		else {
-			this.callbackError("NoActiveConnection");
-		}
-	}
-
-	turnOnLocalCamera() 
-	{
-		//If it started in playOnly mode and wants to turn on the camera
-		if(this.localStream == null){
-			this.navigatorUserMedia(this.mediaConstraints, stream =>{
-				this.gotStream(stream);
-			}, false);
-		}
-		else if (this.remotePeerConnection != null) {
-			var track = this.localStream.getVideoTracks()[0];
-			track.enabled = true;
-		}
-	}
+	turnOffLocalCamera(streamId) 
+	 {
+		 this.dummyCanvas.getContext('2d').fillRect(0, 0, 320, 240);
```

Comment:
I think it should be compatible with older version. Shall we think a way of doing that?